server/tilesource/mapniksource.py

```diff
@@ -232,6 +232,11 @@ def getBounds(self, srs=None):
                 },
                 'srs': nativeSrs,
             }
+            # Make sure geographic coordinates do not exceed their limits
+            if pyproj.Proj(nativeSrs).is_latlong() and srs:
+                yBound = 85.0511 if '3857' in srs else 90.0
```

Comment:
@manthey Do you know a better way to check if srs is web mercator?
```python
if '3857' in srs
```
This is kind of ugly.